### ISSUE TITLE
B009 complains about `getattr` with default value

### ISSUE THREAD

**Issue by inducer** [PyCQA/flake8-bugbear#232]
With version 21.11.29, I get a B009 complaint
```
relate/utils.py:359:23: B009 Do not call getattr with a constant attribute value, it is not any safer than normal property access.
```
for this code:
```py
getattr(settings, "EMAIL_CONNECTION_DEFAULT", None)
```
Subjectively, that seems preferable to `getattr` wrapped in a `try`/`except` to account for the possibility of the attribute not being there. Maybe B009 could be narrowed to only be triggered if no default value is given?

**Comment by jpy-git** [comment]
Interesting, I'm not able to reproduce this.

Looking at the [source](https://github.com/PyCQA/flake8-bugbear/blob/3206da7d3a3ef98f7c2bd38d7b533cb1d65f8a3b/bugbear.py#L309-L315), B009 should only be raised if the `getattr` has 2 args, i.e. no default supplied.

There's also [unit test](https://github.com/PyCQA/flake8-bugbear/blob/3206da7d3a3ef98f7c2bd38d7b533cb1d65f8a3b/tests/b009_b010.py#L9) that pretty much checks this exact example.

**Comment by jpy-git** [comment]
Actually tracking down [line 359](https://github.com/inducer/relate/blob/bcf963f65873f271f7a52bcff335bda5e56bf064/relate/utils.py#L359) in `relate/utils.py` the rule is correctly flagging `getattr(settings, "EMAIL_CONNECTIONS")`.

As the rule says, it's best to use `settings.EMAIL_CONNECTIONS` instead 😄

**Comment by inducer** [comment]
Argh! I was off by a few lines. I'm sorry for the noise. And thanks for the detective work!
